notes.txt

```diff
@@ -0,0 +1,59 @@
+org.maplibre.navigation.android.navigation.v5.utils.RouteUtils --> Now static (object) class.
+
+org.maplibre.navigation.android.navigation.v5.navigation.NavigationLifecycleMonitor --> Removed. Not used internal.
+org.maplibre.navigation.android.navigation.v5.navigation.NavigationMapRoute --> Deprecated/Removed. In UI package a newer version is available.
+org.maplibre.navigation.android.navigation.v5.navigation.SdkVersionChecker --> Deprecated/Removed. Make no sense, use plain java/kotlin code.
+org.maplibre.navigation.android.navigation.v5.navigation.NavigationTimeFormat --> Move to MapLibreNavigationOptions and migrate to enum class.
+
+org.maplibre.navigation.android.navigation.v5.route.MapRouteProgressChangeListener --> Deprecated/Removed. In UI package a newer version is available.
+org.maplibre.navigation.android.navigation.v5.route.OnRouteSelectionChangeListener --> Deprecated/Removed. In UI package a newer version is available.
+
+org.maplibre.navigation.android.navigation.v5.location.MetricsLocation --> Removed. Seems not used.
+org.maplibre.navigation.android.navigation.v5.location.replay.GpxParser --> Removed. Seems not used.
+org.maplibre.navigation.android.navigation.v5.location.replay.ParseGpxTask --> Removed. Seems not used.
+org.maplibre.navigation.android.navigation.v5.location.replay.ReplayJsonRouteDto --> Removed. Seems not used.
+org.maplibre.navigation.android.navigation.v5.location.replay.ReplayJsonRouteLocationMapper --> Removed. Seems not used.
+org.maplibre.navigation.android.navigation.v5.location.replay.ReplayLocationDto --> Removed. Seems not used.
+org.maplibre.navigation.android.navigation.v5.location.replay.TimestampAdapter --> Removed. Seems not used.
+
+org.maplibre.navigation.android.navigation.v5.navigation.camera.RouteInformation --> Remove Builder, use named arguments in constructor instead.
+org.maplibre.navigation.android.navigation.v5.navigation.camera.Camera --> Remove function `target`, because it's not used anywhere.
+
+org.maplibre.navigation.android.navigation.v5.models.RouteOptions --> Remove `*list` functions. Seems not used anywhere.
+
+org.maplibre.navigation.android.navigation.v5.navigation.MapLibreNavigationOptions --> Remove `isFromNavigationUi`, because seems not used anywhere.
+
+----------------
+Refactor later:
+
+org.maplibre.navigation.android.navigation.v5.milestone.TriggerProperty --> Seems too complicated. Create a fresh new re-write.
+org.maplibre.navigation.android.navigation.v5.location.replay.* --> Hard to refactor in clean Kotlin. Also seems too complicated. Create a fresh new re-write.
+Add adapters or use DTOs instead of parsing directly to models that can be replaces by developers to use other API engines
+org.maplibre.navigation.android.navigation.v5.utils.time.* --> Seems too complicated. Create a simpler version.
+org.maplibre.navigation.android.navigation.v5.utils.RouteUtils --> Seems too complicated. Create simpler versions of util functions.
```

Comment:
Yes or maybe allow to overwrite/extend these

Reply:
Ohh.. I need to delete the `notes.txt`, it's the same text like the PR description.